app/controllers/books_controller.rb

```diff
@@ -0,0 +1,11 @@
+# Controller to handle book management
+class BooksController < ApplicationController
+  respond_to :json
```

Comment:
you can move this to ApplicationController instead of repeating it in every controller